src/app/components/home/home.component.ts

```diff
@@ -16,51 +12,32 @@ import {SubSink} from 'subsink';
 
 export class HomeComponent implements OnInit {
   panelOpenState = false;
-  realms: Realm[];
-  currentRealm: Realm;
+  realms = this.realmService.realms$;
+  currentRealm = this.realmService.realm$;
 
   subSink = new SubSink();
 
   constructor(private router: Router,
               private realmService: RealmService,
-              private adminAuthService: AdminAuthService,
-              private roleService: RoleService,
-              private clientService: ClientService,
-              private snackBar: SnackBarService) {
+              private adminAuthService: AdminAuthService) {
+    this.realmService.getRealms().subscribe(realms => {
+      this.realmService.setRealm(realms[0]);
```

Comment:
redenumeste functia in setCurrentRealm, ca sa poti deosebi mai usor de setRealms